kimchi/src/circuits/constraints.rs

```diff
@@ -593,7 +644,10 @@ impl<F: FftField + SquareRootField> ConstraintSystem<F> {
         // ------
 
         let lookup_constraint_system =
-            LookupConstraintSystem::create(&gates, lookup_tables, &domain);
+            match LookupConstraintSystem::create(&gates, lookup_tables, &domain) {
+                Ok(res) => res,
+                Err(_) => None?,
+            };
```

Comment:
note that there's a PR that will make this function return a Result instead of an Option: https://github.com/o1-labs/proof-systems/pull/490

for now, you can use

```rust
let lookup_constraint_system = LookupConstraintSystem::create(&gates, lookup_tables, &domain).ok()?;
```